crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java

```diff
@@ -111,7 +122,7 @@ public Response getSearchesForEntity(@PathParam("entity") String entity,
         }
         CallStatus st=new FindCommand(freq.getEntityVersion().getEntity(),
                                       freq.getEntityVersion().getVersion(),
-                                      freq.toJson().toString()).run();
+                                      freq.toJson().toString(), metrics).run();
```

Comment:
I'm surprised we need to pass in the metrics here. Mediator should already know how to track this; it's now not the responsibility of the HTTP layer to track metrics.

Reply:
@svkm102 Can you do a find/replace on version number to substitute underscore ( _ ) for the dot ( . ) in the version?  Graphite treats . specially so it will throw things off if we pass that along.  So something like `metrics:name=api.find.test.1.0.0.requests.active`
would become
`metrics:name=api.find.test.1_0_0.requests.active`

Reply:
Note that I didn't mean to change it in the command calls, just wherever you are defining the name of the metric being captured

Reply:
Updated the version handling to replace "." with "_" in lightblue-core. That should take care of versions in all commands.